test/utils/isSameDay_spec.js

```diff
@@ -15,6 +15,15 @@ describe('isSameDay', () => {
     expect(isSameDay(today, tomorrow)).to.equal(false);
   });
 
+  it('returns false for same days of week', () => {
+    // Flags accidentally use of moment's day() function, which returns index
```

Comment:
Ah ok, so there's never been a release with "is same day" semantics, only "is same date"?

In that case it's just a bad name for the function, but that can be changed later, and this code looks fine.